projects/fsastorefrontlib/src/cms-components/my-account/quote/quotes/quotes.component.html

```diff
@@ -3,89 +3,179 @@ <h2 class="heading-headline py-3 mt-3 mb-5 text-center">
 </h2>
 <ng-container *ngIf="quotes$ | async as quotes">
   <ng-container *ngIf="quotesLoaded$ | async; else loading">
-    <div class="row" *ngIf="quotes.length > 0; else noQuotes">
-      <div *ngFor="let quote of quotes" class="col-12 col-md-6 d-flex">
-        <div class="info-card">
-          <h6 class="title background-color-4">
-            {{ quote.defaultCategory?.name }}
-          </h6>
-          <div class="info-card-wrapper">
-            <div class="info-card-content">
-              <div class="info-card-details">
-                <div class="info-card-data">
-                  <span class="label">{{ quote.planName }}</span
-                  >:
-                  <span class="value">{{ quote.quoteId }}</span>
-                </div>
-                <ng-container *ngIf="quote.quoteWorkflowStatus">
+    <div *ngIf="quotes.length > 0; else noQuotes">
+      <div class="row justify-content-between align-items-center mb-5">
+        <div class="col-md-5 d-flex">
+          <label class="col-8 pr-3 px-0">
```

Comment:
this all should be wrapped again with `row`

Reply:
just `col-8`

Reply:
> this all should be wrapped again with `row`

Another `row` class will broke everything.